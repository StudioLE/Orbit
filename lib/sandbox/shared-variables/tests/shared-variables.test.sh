#!/bin/bash
set -uo pipefail

# ARGS
ACTUAL_FILE=$1

/srv/lib/sandbox/shared-variables/parent.sh > "${ACTUAL_FILE}"
