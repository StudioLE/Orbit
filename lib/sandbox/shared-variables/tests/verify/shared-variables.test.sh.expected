> PARENT VALUE
> > PARENT VALUE
> > > CHILD VALUE
> > > GRAND CHILD VALUE
> > PARENT VALUE
> PARENT VALUE
